src/environment-variables/index.js

```diff
@@ -4,9 +4,9 @@ const dotenv = require('dotenv');
 const defaultValues = {
   EPISODE_PATH: '.',
   EPISODE_FILE: 'episode.json',
-  ANCHOR_LOGIN: true,
-  ANCHOR_EMAIL: '',
-  ANCHOR_PASSWORD: '',
+  SPOTIFY_LOGIN: true,
```

Comment:
I think that you don't need to add any other environment variables. You can just delete the `ANCHOR_` env variables.

Reply:
I agree